app/frontend/components/pages/delegations/delegatePage.tsx

```diff
@@ -46,58 +49,145 @@ const BigDelegatorOffer = (): h.JSX.Element => (
   </div>
 )
 
+// REFACTOR: (Untyped errors): move to types
+// is "hasTickerMapping" something specific or general?
+type Error = {
+  code: string
+  params?: {hasTickerMapping: boolean}
+}
+
+type ValidatedInput = {
+  poolHash: string | null
+  error: Error
+}
+
+const validateInput = (
+  fieldValue: string,
+  validStakepoolDataProvider: StakepoolDataProvider
+): ValidatedInput => {
+  if (ADALITE_CONFIG.ADALITE_ENABLE_SEARCH_BY_TICKER) {
+    const pool =
+      validStakepoolDataProvider.getPoolInfoByPoolHash(fieldValue) ||
+      validStakepoolDataProvider.getPoolInfoByTicker(fieldValue)
+    if (pool) return {poolHash: pool.poolHash, error: null}
+
+    const hasTickerMapping = validStakepoolDataProvider.hasTickerMapping
+    const isTickerString = fieldValue.length <= 5 && fieldValue.toUpperCase() === fieldValue
```

Comment:
I don't think `fieldValue.toUpperCase() === fieldValue` is necessary, there can be [lower case tickers](https://pooltool.io/pool/76619d4c31c0570824e369d13a95542b65ee29546f2527f96042e602/epochs) as well, cc @PeterBenc

Reply:
Agreed we should remove the `field.toUpporCase()` condition :+1: